config/presets/identity/email-password-provider.json

```diff
@@ -4,8 +4,8 @@
     {
       "@id": "urn:solid-server:default:IdentityProviderHandler",
       "@type": "EmailPasswordIdentityProviderHandler",
-      "EmailPasswordIdentityProviderHandler:_args_oidcProviderFactory": {
-        "@id": "urn:solid-server:default:OidcProviderFactory"
+      "EmailPasswordIdentityProviderHandler:_args_solidIdentityProviderFactory": {
+        "@id": "urn:solid-server:default:SolidIdentityProviderFactory"
```

Comment:
Why did you change this parameter name to "SolidIdenityProviderFactory." It doesn't necessarily need to be a SolidIdentityProvider, it could be an IdentityProvider